### ISSUE TITLE
ng-touched is not being added

### ISSUE THREAD

**Issue by srigiriharish** [NileshPatel17/ng-multiselect-dropdown#47]
While writing validations to check whether the field is empty or not I wrote a condition which shows an error mesage when the field is touched and  empty

But this multiselect drop down is just showing ng-untouched even after it is touched, 

I even checked in the demo that is given here, it is also the same, it remains in untouched state even though it is touched,

**Comment by kvarunsameer** [comment]
Yes am also facing the same issue... did u find any solution? how are you managing this @srigiriharish

**Comment by srigiriharish** [comment]
No, I didn't find any solution. I just used the click event and checked whether the select box is clicked rather than touched @kvarunsameer
